src/components/dynamic/admin/dashboards/Toolbar.jsx

```diff
@@ -134,23 +159,26 @@ const Toolbar = ({
       <div className="flex w-1/3">
         <FaTrashAlt
           data-cy="delete"
-          onClick={() =>
-            setPopup({
-              ...popup,
-              visible: true,
-            })
-          }
+          onClick={() => setDeletePopup({ ...deletePopup, visible: true })}
           size={22.5}
           className="ml-5 text-hackathon-gray-300 hover:opacity-70 duration-150 hover:cursor-pointer"
         />
-        {popup.visible && (
+        {deletePopup.visible && (
```

Comment:
there should only be one popup, we should just be changing the content of the popup

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/122486013/f3b4ed71-1258-47cb-90d8-fd011cc5baf2)
fixed code